src/SplitPackages/Program.cs

```diff
@@ -227,7 +227,7 @@ private ProjectJsonFileBuilder CreateBaseJsonFileBuilderForCache(string destinat
 
             var hardcodedDependencies = new[]
             {
-                new PackageInformation(null, "Microsoft.NetCore.App", "1.0.0", new[] { Frameworks.NetCoreApp10 })
+                new PackageInformation(null, "Microsoft.NetCore.App", "1.1.0", new[] { Frameworks.NetCoreApp10 })
```

Comment:
I wasn't sure if having a `netcoreapp1.0` targeting tool would affect this (those are the only things we ship that target netcoreapp). If this is specifically used to produce a project.json, this should be fine